build.sh

```diff
@@ -225,6 +225,10 @@ while [[ $# -gt 0 ]]; do
         NO_JIT="-DNO_JIT_SH=1"
         ;;
 
+    --intl-icu)
```

Comment:
Yes I'm changing the flag to `--with-intl` (definitely not making it the default at this time, maybe never will, or in any case it will be optional) and I'm keeping the macro named `INTL_ICU` for consistency with Windows and so I don't have to churn that at this time.